k8s/crds/kops.k8s.io_clusters.yaml

```diff
@@ -1867,6 +1867,10 @@ spec:
                     description: CoreDNSImage is used to override the default image
                       used for CoreDNS
                     type: string
+                  cpaImage:
```

Comment:
I'm not sure this image override field should live under the `kubeDNS` spec

Reply:
the image is only used by coredns and kubedns... which is why I kept it there.

Reply:
I am open to putting it elsewhere... under which object would you suggest I put it?

Reply:
Yeah I guess I was thinking since its more a general purpose autoscaler than _just_ DNS but if its only used for DNS out of the box in a new cluster then seems fine for it be here!